src/ocr/ocr.cpp

```diff
@@ -83,6 +83,12 @@ void Ocr::recognize(const QPixmap &pixmap, int dpi)
     m_future = QtConcurrent::run([this, dpi, image = pixmap.toImage()] {
         m_tesseract.SetImage(image.constBits(), image.width(), image.height(), image.depth() / 8, image.bytesPerLine());
         m_tesseract.SetSourceResolution(dpi);
+        QMap<QString, QVariant> parameters = AppSettings().tesseractParameters();
+        for (auto it = parameters.cbegin(); it != parameters.cend(); ++it) {
+            if (!m_tesseract.SetVariable(qPrintable(it.key()), qPrintable(it.value().toString()))) {
+                qWarning() << tr("%1 is not the name of a valid tesseract parameter.").arg(it.key());
+            }
```

Comment:
[qPrintable](https://doc.qt.io/qt-5/qvariant.html#toByteArray) is the same as `str.toLocal8Bit().constData()`. But in this case `str.toLocal8Bit()` is enough. Also `QVariant` can be transformed directly to `QByteArray`.
Also, I use `{}` only for multiline things. This is a part of the official Qt Coding Style. I made a suggestions for all other places.
```suggestion
            if (!m_tesseract.SetVariable(it.key().toLocal8Bit(), it.value().toByteArray()))
                qWarning() << tr("%1 is not the name of a valid tesseract parameter.").arg(it.key());
```